src/components/sbb-calendar/sbb-calendar.scss

```diff
@@ -71,25 +71,29 @@
 }
 
 .sbb-calendar__table-data {
-  position: relative;
+  padding: var(--sbb-calendar-day-padding);
```

Comment:
seems to introduce a new bug on placing the strike through on filtered out dates.
<img width="866" alt="Bildschirmfoto 2023-04-12 um 08 02 58" src="https://user-images.githubusercontent.com/29278232/231365867-661806a1-924a-481e-ae36-1497ed2de4f7.png">